docs/DeveloperGuide.md

```diff
@@ -574,6 +574,20 @@ The sequence diagram is as follows:
 
 The multiple commands executed will return a `CommandResult` which contains a list of feedback messages of the results executed combined all together and returned as `feedbackToUser`
 
+### \[Not implemented\] Tag Finding Feature
```

Comment:
I suggest changing to `Proposed` instead of `Not Implemented`.

Reply:
Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/337/commits/076d80a01e9d74a518c808868b5285ed5a23d150